README.rst

```diff
@@ -13,37 +13,42 @@ Micro-frontend for the single-page payment/checkout process. This application su
 Getting Started
```

Comment:
Above in the intro, I might add a note about the domain - ie that order history is in its own mfe (I think I have that right?), and anything else checkout related that's explicitly not included

Reply:
There is an order history MFE but it is not yet enabled by default; however, yes, it would be good to mention that this frontend is _just_ checkout, and that it is _extremely_ Cybersource-specific

Reply:
Agreed, that would be a good addition